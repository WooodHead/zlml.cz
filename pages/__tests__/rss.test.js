// @flow

import Rss from '../rss';

it('is not component', () => {
  expect(Rss()).toBe(undefined);
});

it('sends expected headers to the client', () => {
  expect.assertions(4);
  Rss.getInitialProps({
    res: {
      writeHead: (status, headers) => {
        expect(status).toBe(200);
        expect(headers).toMatchSnapshot();
      },
      write: body => {
        expect(body).toBeTruthy();
      },
      end: () => {
        expect(true).toBe(true); // expect it's called
      },
    },
  });
});
