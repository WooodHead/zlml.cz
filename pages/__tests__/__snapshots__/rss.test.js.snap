// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sends expected headers to the client 1`] = `
Object {
  "Content-Type": "application/xml; charset=utf-8",
}
`;
